diff --git a/out/getProgram.js b/out/getProgram.js
index bc8074e09d33e13cd158f2fd29c83b4a6fed865e..591636448178050aecceb01024501bdc20cd3eb9 100644
--- a/out/getProgram.js
+++ b/out/getProgram.js
@@ -21,12 +21,11 @@ function getProgram(ts, core, ls) {
             }
             return target[property];
         },
-        // commented out because it causes https://github.com/vuejs/language-tools/issues/2403
-        // set: (_target, property, newValue) => {
-        // 	const program = getProgram();
-        // 	(program as any)[property] = newValue;
-        // 	return true;
-        // },
+        set: (target, property, newValue) => {
+        	const program = getProgram();
+        	target[property] = program[property] = newValue;
+        	return true;
+        },
     });
     function getProgram() {
         return ls.getProgram();
