diff --git a/out/getProgram.js b/out/getProgram.js
index bc8074e09d33e13cd158f2fd29c83b4a6fed865e..3ede7246a0c29b162a249622515f90f49c289856 100644
--- a/out/getProgram.js
+++ b/out/getProgram.js
@@ -22,11 +22,10 @@ function getProgram(ts, core, ls) {
             return target[property];
         },
         // commented out because it causes https://github.com/vuejs/language-tools/issues/2403
-        // set: (_target, property, newValue) => {
-        // 	const program = getProgram();
-        // 	(program as any)[property] = newValue;
-        // 	return true;
-        // },
+        set: (_target, property, newValue) => {
+        	getProgram()[property] = newValue;
+        	return true;
+        },
     });
     function getProgram() {
         return ls.getProgram();
